src/debugging/coreclr/dockerDebugConfigurationProvider.ts

```diff
@@ -78,7 +79,13 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
                 dockerBuild: {
                 },
                 dockerRun: {
-                }
+                    env: {
```

Comment:
>env [](start = 20, length = 3)

Does this need to be limited to aspnetcore only?

Reply:
TBH I don't know what this method is for, because it's not actually the source of the debug configuration (which comes from package.json)